package.json

```diff
@@ -20,8 +20,8 @@
     "posttest": "gulp build",
     "start": "nodemon -e html,js ./index.js",
     "lint": "gulp lint",
-    "ava:watch": "NODE_CONFIG_DIR=../config ava --watch | tap-diff",
-    "ava": "NODE_CONFIG_DIR=../config ava | tap-diff",
+    "ava:watch": "NODE_ENV=default-test NODE_CONFIG_DIR=../config ava --watch | tap-diff",
```

Comment:
This _should_ just be "test" and maybe we rename the test configuration to go with it